js1.js

```diff
@@ -0,0 +1,12 @@
+"use script"
+
+let a=5;
+let b=6;
+let c=9;
+let P=a+b+c; //Search perimeter of triangle
```

Comment:
for the name of variable use camelCase and use more relevant names. For example perimeterOfTriangle